src/__experimental__/components/grouped-character/grouped-character.component.js

```diff
@@ -27,7 +39,10 @@ const GroupedCharacter = (props) => {
 
     const reProcessedInputValue = ev.target.value.split(separator).join('');
 
-    props.onChange({ target: { value: reProcessedInputValue } });
+    onChange({ target: { value: reProcessedInputValue } });
```

Comment:
We need to include the whole `event` in the callback e.g
```js
const target = {
  ...ev.target,
value: reProcessedInputValue
}

ev.target = target;
onChange(ev);
```

Reply:
How about:
```js
const event= {
  ...ev,
  target: {
    ...ev.target,
    value: reProcessedInputValue
  }
}
onChange(event);
```
No mutations and imo looks more readable.